test/simulation.jl

```diff
@@ -7,13 +7,13 @@
     gen_temp_wrapper(s, args...; kwargs...) = temperature(s)
 
     s = System(
-        atoms=[Atom(mass=10.0u"g/mol", charge=0.0, σ=0.3u"nm", ϵ=0.2u"kJ * mol^-1") for i in 1:n_atoms],
-        coords=place_atoms(n_atoms, boundary; min_dist=0.3u"nm"),
+        atoms=CuArray([Atom(mass=10.0u"g/mol", charge=0.0, σ=0.3u"nm", ϵ=0.2u"kJ * mol^-1") for i in 1:n_atoms]),
```

Comment:
Have a `if run_gpu_tests` block that adds GPU tests below the CPU tests. Currently this is causing the CI tests to fail as CUDA is not available there.